core/templates/pages/welcome-guide-page/welcome-guide-page.controller.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controllers for the Welcome Guide page.
+ */
+
+require('domain/utilities/url-interpolation.service.ts');
+
+angular.module('oppia').directive('welcomeGuidePage', [
```

Comment:
Can you please change it to a component? You can check some examples in this PR: #9317, please let me know if you face any issues!